stdlib/public/SDK/CoreFoundation/CoreFoundation.swift

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Unfortunately our CMake setup currently takes the name of the result file from the name of the first file in the module.